dbt_subprojects/daily_spellbook/models/_projects/gmx/event/arbitrum/gmx_v2_arbitrum_event_schema.yml

```diff
@@ -0,0 +1,149 @@
+version: 2
+
+models:
+  - name: gmx_v2_arbitrum_OrderCreated
```

Comment:
```suggestion
  - name: gmx_v2_arbitrum_order_created
```

Reply:
similar to other comment, update file name of model too to match

Reply:
Done